ting.client.inc

```diff
@@ -226,6 +226,16 @@ function ting_get_objects($ids) {
  */
 function ting_do_search($query, $page = 1, $results_per_page = 10, $options = array()) {
   $request = ting_get_request_factory()->getSearchRequest();
+
+  $query = _ting_search_quote($query);
```

Comment:
As mentioned by @xendk `_ting_search_quote` should not be called here. Please remove this line. Here we expect the query to be valid.